query/view/focus_multi_match.js

```diff
@@ -0,0 +1,16 @@
+const peliasQuery = require('pelias-query');
+
+module.exports = (view_name) => (vs) => {
+  const input = vs.var(`multi_match:${view_name}:input`).get();
+
+  if (!input || input.length < 1) {
+    return null;
+  }
+
+  return {
+    'function_score': {
+      'query': peliasQuery.view.leaf.multi_match(view_name)(vs),
+      'score_mode': 'multiply',
+    },
+  };
```

Comment:
question about this...

what's the purpose of wrapping the 'leaf' query in a 'function_score' here?
doesn't the 'function_score' query require an Array 'functions', with at least one function.
the 'score_mode' property then effects how scoring is applied to the multiple 'functions'

Reply:
I guess what I'm asking is, what's the difference between this and:

```
return peliasQuery.view.leaf.multi_match(view_name)(vs)
```

Reply:
I tried different `score_mode` and wanted to be consistent with other functions, but you're right, since I'm using the defaults value, it's semantically the same, so I can return the leaf instead of the `function_score` :+1: